app/src/main/kotlin/studio/lunabee/compose/demo/glance/GlanceScreen.kt

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright © 2022 Lunabee Studio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * TextScreen.kt
+ * Lunabee Compose
+ *
+ * Created by Lunabee Studio / Date - 11/30/2022 - for the Lunabee Compose library.
+ */
+
+package studio.lunabee.compose.demo.glance
+
+import android.content.Context
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Button
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.unit.dp
+import kotlinx.coroutines.Dispatchers
+import studio.lunabee.compose.glance.helpers.PinWidgetToHomeScreenHelper
+
+@Composable
+fun GlanceScreen(
+    context: Context = LocalContext.current,
+) {
+    val pinWidgetHelper: PinWidgetToHomeScreenHelper = remember {
+        PinWidgetToHomeScreenHelper(context, Dispatchers.IO)
+    }
+
+    Box(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(all = 16.dp),
+    ) {
+        if (pinWidgetHelper.isPinSupported()) {
+            Button(
+                onClick = {
+                    pinWidgetHelper.pin(GlanceWidgetDemo::class.java, GlanceWidgetDemoReceiver::class.java)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/LunabeeStudio/Lunabee_Compose_Android/blob/6c09cd48d00137585ef42b09da4d94f399394e06/app/src/main/kotlin/studio/lunabee/compose/demo/glance/GlanceScreen.kt#L54"></span>Call requires API level 26 (current min is 21): <code>pin</code></td>
    </tr>
  </tbody>
</table>

Reply:
Fait un effort en fait